pkgs/misc/emulators/libdsk/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl }:
+
+with stdenv.lib;
```

Comment:
Probably move this to meta, since you're not using any of stdenv.lib elsewhere and it may cause confusion